ingredient/ingredient-functions/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import { FunctionsPlugin} from './plugin'
+//import { FunctionsUtils} from './functions'
+
+/*  comment out these entries if you are not including an ingredient plugin runner*/
+exports.plugin = FunctionsPlugin
+exports.pluginNS = "@azbake/ingredient-functions" //name of the ingredient to reference in a bake.yaml recipe
+
+/* comment out these entries if you are not including a set of expression functions*/
```

Comment:
For this ingredient, you'll want a util function with at least one function for generating the function app name. Check something like the storage ingredient for how we do that.